docs/reference/modules/network.asciidoc

```diff
@@ -163,18 +163,18 @@ The size of the TCP receive buffer (specified with <<size-units,size units>>).
 By default not explicitly set.
 
 [discrete]
-=== Transport and HTTP protocols
+=== HTTP and transport network layers
 
-An Elasticsearch node exposes two network protocols which inherit the above
-settings, but may be further configured independently:
+An {es} node has two network layers which inherit the above settings:
 
-TCP Transport::
+HTTP network layer::
 
-Used for communication between nodes in the cluster, by the Java
-{javaclient}/transport-client.html[Transport client].
-See <<modules-transport>>.
+The <<modules-http,HTTP network layer>> exposes the JSON-over-HTTP interface
+used by clients.
 
-HTTP::
+Transport network layer::
```

Comment:
I'm not sure I like the "layer" naming (neither no I like that we refer to modules). Can we just avoid these terms and sub-sectioning altogether and talk directly about what is being configured where? For example, "Clients should typically communicate with Elasticsearch via <<modules-http,HTTP>>, while Elasticsearch nodes are internally using an internal protocol for inter-node communication, ...  Both HTTP and the internal protocol inherit the above settings, but can also be separately configured.

Reply:
We need some kind of noun but I've tried to avoid it as much as possible in 919c84e. The fact that there are these two modules/layers/interfaces/whatever which are both subsidiary to the network thing we're discussing here is not well-understood and yet quite important for getting how it all fits together. I'd rather keep some formatting that makes it clear that there's these two separate things, although I've weakened it to a bulleted list.